src/Function/Properties/Inverse.agda

```diff
@@ -48,6 +49,9 @@ isEquivalence = record
   ; trans = Composition.inverse
   }
 
+open module ↔ {ℓ} = IsEquivalence (↔-isEquivalence {ℓ})
```

Comment:
We need to add a `using ()` clause otherwise we're unintentionally exporting a load of other junk from this module as well. Can you also quickly check the module `↔` isn't exposed outside of `Function.Properties.Inverse`? Can't remember if it is or not...

Reply:
I have already put `using ()`.